pbm/backup/oplog.go

```diff
@@ -121,22 +118,17 @@ func (ot *Oplog) WriteTo(w io.Writer) (int64, error) {
 	return written, cur.Err()
 }
 
-// LastWrite returns a timestamp of the last write operation readable by majority reads
-func (ot *Oplog) LastWrite() (primitive.Timestamp, error) {
-	return pbm.LastWrite(ot.node.Session(), true)
-}
-
-func (ot *Oplog) collectionName() (string, error) {
-	inf, err := ot.node.GetInfo()
+// IsSufficient check is oplog is sufficient back from the given date
+func (ot *Oplog) IsSufficient(from primitive.Timestamp) (bool, error) {
+	c, err := ot.cl.CountDocuments(context.Background(), bson.M{"ts": bson.M{"$lte": from}}, options.Count().SetLimit(1))
```

Comment:
I thought this might be slow due to possibly doing a collscan from the new end backwards.

But I ran `db.oplog.rs.explain().count({"ts": {$lte: from}}, {limit: 1})` and it seems to be no problem ... it's a collscan, but from the old end. So works nicely for the $lt/$lte case.